handbook/engineering/engineering-management.md

```diff
@@ -49,20 +49,52 @@ Here are some books that multiple people on the team have read and recommend. If
+To be documented.
 
-### Interim Manager
+## Temporary management positions
 
-Interim Manager positions are for teammates pursuing an Engineering Manager role long term but lack the necessary people management experience. The requirement is that, before moving into the role full time, they will make at least one successful hire. The official transition to Engineering Manager will not occur before 30 days after that new hire's start date, so that we can assess whether the hire was truly successful. If the new hire's success is indeterminate at the 30-day mark, we will continue to review until a firm decision is made. If the new hire is not successful, that does not mean that the Interim Manager cannot eventually move into the full-time role.
+We create temporary management positions when there is an organizational need. Some examples:
 
-In some cases, it may not be practical, or headcount planning might not allow for the Interim Manager to make a new hire. In this case, the Interim Manager and the interim manager's manager should agree on success criteria based on the role's requirements.
+- A team has grown beyond the [maximum viable team guideline](eng_org.md#maximum-viable-team), and we decide to split the team.
+- An Engineering Manager is out on extended PTO.
+- We decide to form a new team from scratch. 
 
-Once the Interim Manager's first new hire has been at Sourcegraph for 30 days or other agreed success criteria have been met, their manager will formally review the Interim Manager.
+### Acting manager
 
-This process aims to have a determination made by the interim manager's manager about the promotion to a permanent manager role after the agreed success criteria have been satisfied. Generally, we aim to have the interim period not exceed four months before turning into a full-time position.
+An Acting Manager occupies a management role temporarily and will move back to their original position after a set amount of time or other conditions. An Acting Manager could be filling in for a vacant role while we hire or providing cover for another Engineering Manager on extended PTO.
+
+### Manager in training
+
+Manager in training positions are for teammates pursuing an Engineering Manager role long term but who lack the necessary people management experience. We want to allow teammates to consider if they would like to make the role switch and verify that they would be good in the role.
+
+#### Requirements for a manager in training role
+
+The following requirements are in place to ensure we set up the manager in training for success.
+
+1. The candidate's manager or another nominated manager must commit to providing extra support to the manager in training for the duration of the trial period. They become the sponsoring manager.
```

Comment:
> or another nominated manager

It feels a bit risky to me if the MIT is going to someone other than their sponsoring manager for advice. These conversations are important/useful for calibrating throughout the process

Reply:
The intention here was to say that the IC's line manager OR another manager (say a Director or EM in another org) can be the sponsoring manager. I'll rephrase it.

```suggestion
1. The candidate's manager or another nominated manager will become the sponsoring manager. The sponsoring manager must commit to providing extra support to the manager in training for the duration of the trial period.
```